### PR TITLE
new track panel config

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1255

## Importance
Not on sprint 14 goal, so not critical

## Description
Update the current BrowserTrackConfig component to bring it closer to the new design:

## Deployment URL
http://new-track-panel-config.review.ensembl.org/genome-browser/homo_sapiens_GCA_000001405_14?focus=region:17:64155265-64255266&location=17:64155265-64255266

## Views affected
GB

### Other effects
N/A

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
NA

### CODE REVIEW

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.scss [hunk 1]
```diff
@@ -46,31 +42,9 @@
...
-  .trackMoveIcon {
-    height: 28px;
-    width: 14px;
-  }
-}
-
-.checkboxHolder {
-  width: 100%;
-}
+}
```
Comment (jyothishnt) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.scss:50]: no new line :-)

File: src/ensembl/src/content/app/browser/browser-cog/BrowserCog.tsx [hunk 2]
```diff
@@ -93,12 +94,16 @@ const BrowserCog = (props: BrowserCogProps) => {
...
-          status={getCogIconStatus()}
-          description={cogIconConfig.description}
-          onClick={toggleCog}
-          image={cogIconConfig.icon}
-        />
+        {cogActivated ? (
+          <CloseButton onClick={toggleCog} />
+        ) : (
+          <ImageButton
+            status={getCogIconStatus()}
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browser-cog/BrowserCog.tsx:101]: Check the `getCogIconStatus` function: I suspect it is no longer necessary.

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.scss [hunk 3]
```diff
@@ -2,37 +2,33 @@
 
 .trackConfig {
   display: grid;
-  grid-template-columns: 146px 343px 1fr;
+  grid-template-columns: 140px 250px 1fr;
   background: #f1f2f4;
   box-shadow: 1px 1px 10px #6f8190;
   position: absolute;
-  left: -614px;
+  left: -395px;
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.scss:9]: `left` with the value equal to the width of the panel in pixels was a weird way to define the position. Try `right` to just define the width of the offset, with no concern about the width of the config panel itself.

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.test.tsx [hunk 4]
```diff
@@ -47,13 +47,16 @@ describe('<BrowserTrackConfig />', () => {
   };
 
   describe('behaviour', () => {
-    it('sets all tracks to be updated when the all tracks checkbox is selected', () => {
+    it('sets all tracks to be updated when the all tracks is selected', () => {
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.test.tsx:50]: there's a problem with `the all tracks is` 🙂 

Maybe rephrase the whole test description to `it('can update all tracks', () => {...})`?

**Review by azangru — APPROVED** [review]
